src/main/java/com/webperside/courseerpbackend/models/mybatis/appConfigs/AppConfigs.java

```diff
@@ -0,0 +1,16 @@
+package com.webperside.courseerpbackend.models.mybatis.appConfigs;
+
+import com.webperside.courseerpbackend.models.mybatis.base.BaseEntity;
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class AppConfigs extends BaseEntity<Long> {
```

Comment:
it would be great to name it as AppConfig, and also be sure you are renamed all related classes

Reply:
We use app-configs naming in migration file. Can I also change migration file like app-config?

Reply:
no, we don't need that. 
It's only for entity models. let me clarify that, if we store every app config data on each row on app_configs table, it means we'll get config data in this entity. it would be nice to name it like AppConfig. because every entity will contain only one row, and it indicates that, this app config entity only for one config value